app/views/coronavirus_landing_page/components/shared/_announcements_section.html.erb

```diff
@@ -24,4 +22,5 @@
       </a>
     </p>
   <% end %>
+  <%= render partial: 'coronavirus_landing_page/components/landing_page/risk_level_section', locals: { data: details.risk_level } if details.risk_level %>
```

Comment:
Once we've added the key to the yaml validation specs on govuk-coronavirus-content we won't need the check here. But for now I guess it's sensible as things are a bit fluid!

Reply:
**:nod:**